app/src/main/java/com/example/calculatorapp/fragments/todoapp/TodoFragment.kt

```diff
@@ -0,0 +1,227 @@
+package com.example.calculatorapp.fragments.todoapp
+
+import android.app.*
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.EditText
+import android.widget.LinearLayout
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.calculatorapp.*
+import com.example.calculatorapp.NotificationReceiver.Companion.MESSAGE_EXTRA
+import com.example.calculatorapp.NotificationReceiver.Companion.TITLE_EXTRA
+import com.example.calculatorapp.fragments.FileUltis
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import java.util.*
+
+class TodoFragment : Fragment(R.layout.fragment_todo) {
```

Comment:
toàn bộ warning của file này em xem lại và yêu cầu fix toàn bộ nhé, cái nào có vấn đề thì nêu ra ae bàn bạc thêm